kiruidavid/SendIT254
# #162004444 fix app issues

#### What does this PR do? 
It fixed the app issues making it up to date and meet the feedback expectations 
#### Description of Task to be completed? 
all the issues with the endpoints meet the what was expected 
#### How should this be manually tested? 
clone this repo activate the virtualenv venv and use pytest to see if the tests work and flask run with the respective urls for them to run well.

#### What are the relevant pivotal tracker stories? 
#162004444
